pkg/mkimage-raw-efi/initramfs-init

```diff
@@ -473,24 +474,31 @@ ln -s /proc/mounts /etc/mtab
 # check if setup_loops=... was set
 if [ -n "$KOPT_setup_loops" ]; then
 	# locate boot media and mount it
-	ebegin "Mounting boot media"
-	nlplug-findfs $cryptopts -p /sbin/mdev ${KOPT_debug_init:+-d} \
-		${KOPT_usbdelay:+-t $(( $KOPT_usbdelay * 1000 ))} \
-		-n -r /tmp/rootfs
-	eend $?
-
-        # see if we found anything
-        if [ ! -e /tmp/rootfs ]; then
-           echo "Failed to find rootfs (which is weird). Try to re-run nlplug-findfs manually to see what's wrong:"
+	# NOTE that we may require up to 3 tries with
+	# 30 seconds pauses between them to accomodate
+	# really slow controllers (such as bad USB sticks)
+	for i in 1 2 3; do
+           ebegin "Attempt $i to find and mount boot media"
+           nlplug-findfs $cryptopts -p /sbin/mdev ${KOPT_debug_init:+-d} \
+                ${KOPT_usbdelay:+-t $(( $KOPT_usbdelay * 1000 ))} \
+                -n -r /tmp/rootfs
+	   eend $?
```

Comment:
Is there some identation issue on this line or in this block?

Reply:
I am afraid this is tabs vs. spaces that messes up GH diff formatting -- the new code I write I try to make yetus happy (hence spaces) but the old code had tabs in a few places. Not sure it is worth it to reformat everything -- but I understand how it makes it a bit more difficult to review @eriknordmark

Reply:
Just looked wierd. Might make sense to fix the whole file (as a separate PR or this one).